src/ui/help/index.html

```diff
@@ -35,16 +35,17 @@ <h1> Site-specific info </h1>
 
 <h1> General info about Button Men </h1>
 
-<p> <a href="http://www.cheapass.com/node/39">Button Men</a> is copyright
+<p> <a href="https://cheapass.com/free-games/button-men/">Button Men</a> is copyright
 1999, 2016 James Ernest and Cheapass Games
-(<a href="http://www.cheapass.com/">www.cheapass.com</a> and
-<a href="http://www.beatpeopleup.com">www.beatpeopleup.com</a>),
+(<a href="https://cheapass.com">www.cheapass.com</a> and
+<a href="https://beatpeopleup.cheapass.com">www.beatpeopleup.com</a>),
 and is used with permission. Other <a href="terms.html">terms and conditions</a>
 may be forthcoming. </p>
 
-<p> Longtime online Button Men player Spindisc has
-<a href="http://www.randomdice.com/games/buttonmen/">a site with lots of
-general info about the game</a>. </p>
+<!-- currently returns 404 -->
+<!--<p> Longtime online Button Men player Spindisc has-->
+<!--<a href="http://www.randomdice.com/games/buttonmen/">a site with lots of-->
+<!--general info about the game</a>. </p>-->
```

Comment:
IMO it's unnecessary to comment this out rather than delete it --- it's git, so it'll be in the history.  I don't feel strongly though.